### ISSUE TITLE
LTL generation doesn't handle 1-region maps correctly

### ISSUE THREAD

**Issue by cfinucane** [VerifiableRobotics/LTLMoP#24]
log_2(1) = 0

**Comment by ConstantineLignos** [comment]
Yes! Remember a long time ago when I told you I had fixed some bug in the bit encoding system but couldn't remember what...this is it! (In our frontier-based exploration mode, we ran into this.)
